WordPress/src/main/java/org/wordpress/android/ui/main/MainActionListItem.kt

```diff
@@ -7,6 +7,7 @@ sealed class MainActionListItem {
     abstract val actionType: ActionType
 
     enum class ActionType {
+        NO_ACTION,
```

Comment:
Out of curiosity: Have you considered creating a different child of the sealed class instead of reusing `CreateAction`. It seems we are losing some of the advantages of using a sealed class by setting the values to "null/0/NO_ACTION". Moreover, it feels a bit weird to have CreateAction which doesn't perform any actions. Wdyt?

Reply:
yep, used this approach in the reader bottom sheet that's more complex one (with item divider, tags, sites type etc). I just tried to stay simple here and it worked fine when having only Actions 😊. I don't think this code is final anyway since I expect we will come to the bottom sheet as Chris say so if you agree for now I would keep this one, let me know 🙇‍♂️